README.md

```diff
@@ -1,4 +1,6 @@
 # Benchmark-Goal-15-Coding-Exercises-round-2
+## Team Name: responsible-duck
+## Team Member(s): Jacklyne Bouier
 ## Description
 Benchmark goals are projects that are designed to be done solo. 
```

Comment:
I would add all the specs from the project so that someone unfamiliar knows what the Benchmark goal is about. You can copy past from the original goal in to your own `README`